pkg/controller/podautoscaler/config/v1alpha1/defaults.go

```diff
@@ -34,6 +34,9 @@ import (
 // run it in your wrapper struct of this type in its `SetDefaults_` method.
 func RecommendedDefaultHPAControllerConfiguration(obj *kubectrlmgrconfigv1alpha1.HPAControllerConfiguration) {
 	zero := metav1.Duration{}
+	if obj.ConcurrentHorizontalPodAutoscalerSyncs == 0 {
+		obj.ConcurrentHorizontalPodAutoscalerSyncs = 5
```

Comment:
I would suggest using 1 as default number of workers because it's the current value. If a user needs to increase worker count they can pass the value explicitly.

Reply:
We discussed this on SIG Autoscaling Meeting some time ago and there was an agreement that we should default to 5. Again, the rest of the k8s controllers already default to >1 (mostly 5).